### ISSUE TITLE
"View as Visitor" allows you to take a quiz

### ISSUE THREAD

**Issue by thomasplevy** [gocodebox/lifterlms#694]
### 1. Reproduction Steps

+ Add a quiz to a FREE lesson
+ Use "View as Visitor" from admin bar

### 2. Expected Behavior

+ The quiz should NOT be accessible because quizzes are not accessible to logged out users (even on free lessons)

### 3. Actual Behavior

The quiz is accessible when viewing as a visitor

### 4. Solution

View as functionality should be modified to show NO quiz button to reflect the actual user experience of a visitor / logged out user. Currently it's showing the quiz button because there's no logic built out to hide it

**Comment by LiamBailey** [comment]
I would like to take this if possible?

**Comment by thomasplevy** [comment]
All your @LiamBailey!

If you have any questions please let me know or join us in slack on the `#developers` channel if you want some realtime problem solving: https://lifterlms.com/slack

**Comment by LiamBailey** [comment]
I have made a pull request: https://github.com/gocodebox/lifterlms/pull/705 - please email me on info[at]webbyscots.com if you have a demo site for testing. If not I will set up my own and test. Thanks.

**Comment by thomasplevy** [comment]
@LiamBailey we don't host public testing sites. If you don't already have a way to test WordPress and WordPress plugins locally you can spool up a free temp site using http://poopy.life/ & then install your modified version of LifterLMS for testing

**Comment by LiamBailey** [comment]
Yes sorry for the delay. I will get a test site up and get this tested. Thanks.
